challenges/7-jokenpo-nerd/javascript/michelwene/7-jokenpoNerd.js

```diff
@@ -0,0 +1,44 @@
+const myArgument = process.argv.slice(2);
+
+const numberOfCases = myArgument[0];
+
+const tests = Array.from({ length: numberOfCases }, (_, i) => {
+  const [player1, player2] = myArgument[i + 1].split(" ");
```

Comment:
Tive erro nesta linha ao rodar o comando `node 7-jokenpoNerd.js 3`: TypeError: Cannot read properties of undefined (reading 'split')